src/parsers/index.js

```diff
@@ -0,0 +1,20 @@
+import cucumberParser from './cucumber'
+import multiTestParser from './multitest'
+import standardParser from './standard'
+
+let all = [cucumberParser, multiTestParser, standardParser]
+
+function getParser (name, output = '') {
+  let parser
+  if (name) {
+    parser = all.find((parser) => name === parser.name)
+    if (!parser) { throw new Error(`Invalid Parser Specified: ${name}`) }
+  } else {
+    parser = all.find((p) => {
+      return p.test(output)
+    })
```

Comment:
Great points! I think i'm in agreement on this; it's trying too hard to make things easy. Most people who are going to be using this will know what kind of output they will be using and it's not too heavy to set this up once.

I think 1. is the best and implemented the following way: 

1. Remove all test detection
2. Default `parser` option to `standard`
3. Allow users to select a parser with the `parser` option
4. Document the following in the README

⭐ bonus⭐  
Allow users to specify their own parser module which protractor-flake will require and use?